package.json

```diff
@@ -89,7 +89,7 @@
     "d3-shape": "^3.0.1",
     "d3-time": "^3.0.0",
     "ellipsize": "^0.5.1",
-    "framer-motion": "^11.0.6",
+    "framer-motion": "^10.16.16",
```

Comment:
We might have to roll back rdk for this as well otherwise you end up w/ multiple versions.